resources/definitions/Federation.graphql

```diff
@@ -1,12 +1,16 @@
 # https://www.apollographql.com/docs/federation/subgraph-spec/
-# Latest update - Fed 2.3
+# Latest update - Fed 2.7
 
-# a union of all types that use the @key directive
+# ⚠️ This definition must be created dynamically! The union
+#   must include every object type in the schema that uses
+#   the @key directive (i.e., all federated entities).
 union _Entity
 
 scalar _Any
 scalar FieldSet
 scalar link__Import
+scalar federation__Scope
```

Comment:
In change notes this is declared as `Scope`, not `federation__Scope`. This is confusing because `federation__Policy` is correct. Probably, it should be just `Scope`?

```
directive @requiresScopes(scopes: [[Scope!]!]!) on
    FIELD_DEFINITION
  | OBJECT
  | INTERFACE
  | SCALAR
  | ENUM
```

Reply:
Those changes notes should probably be updated. If you do not import it, the `federation__` prefix is technically on all the directives

https://www.apollographql.com/docs/federation/subgraph-spec#subgraph-schema-additions

We shouldn't have users redeclaring this scalar so really any one works here. I can remove the prefix but this is the full spec additions: https://www.apollographql.com/docs/federation/subgraph-spec#subgraph-schema-additions

Reply:
Thank you for the clarification and the links provided, everything is ok then. I'll merge this to our monorepo today.